Pod/Classes/CollectionViewHelpers.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CollectionViewHelpers.swift
+//  Pods
+//
+//  Created by Jason Grandelli on 12/6/16.
+//
+//
+
+import Foundation
+
+extension UICollectionView {
+    public final func registerReusableCell<T>(_ cell: ReusableCell<T>) where T: UICollectionViewCell {
```

Comment:
This is awesome. I've been meaning to look into this for the tableview equivalent, but is there a way we can use variadic parameters to register multiple cells via a one-line call?